templates/rhpam73-authoring-ha.yaml

```diff
@@ -531,14 +553,14 @@ objects:
 - kind: RoleBinding
   apiVersion: v1
   metadata:
-     name: "${APPLICATION_NAME}-rhpamsvc-view"
+     name: "${APPLICATION_NAME}-rhpamsvc-edit"
      labels:
        application: "${APPLICATION_NAME}"
   subjects:
   - kind: ServiceAccount
     name: "${APPLICATION_NAME}-rhpamsvc"
   roleRef:
-    name: view
+    name: edit
```

Comment:
is it needed to create the config maps?

Reply:
Yes. It is required for KIE server pod updating CM.